src/tweenable.rs

```diff
@@ -446,7 +446,7 @@ impl<T: 'static> Tween<T> {
     /// # Example
     /// ```
     /// # use bevy_tweening::{lens::*, *};
-    /// # use bevy::math::*;
+    /// # use bevy::math::{*,curve::EaseFunction};
```

Comment:
Could you please format all these doc changes (i.e. adding a space after each comma, etc.)?

Reply:
OK